### ISSUE TITLE
Share Management copy button not working on WebKit browsers: Safari / iOS

### ISSUE THREAD

**Issue by MahmoudAlyuDeen** [filebrowser/filebrowser#3534]
### Description
Button does nothing.

### Expected behavior
Copy button works.

### Actual behavior
Button does nothing, no error is given.

### Additional context
Https ✅
Remotely accessible ✅
Copying link works on Chrome ✅
Copying from Safari / iOS ❌

### How to reproduce?

https://github.com/user-attachments/assets/83d0a82d-c930-40c2-94ed-55f28ce36504

**Comment by Johnwulp** [comment]
I have the same issue in Safari version 18.0.1 (20619.1.26.31.7)

**Comment by Toucanto** [comment]
still an issue

**Comment by MahmoudAlyuDeen** [comment]
Stalebot is an enemy of the people...

**Comment by csuporzoltan** [comment]
it does not work on safari macOS. it does the same thing.

**Comment by MahmoudAlyuDeen** [comment]
> it does not work on safari macOS. it does the same thing.

True.

This means that it's a more general problem; since all iOS browsers are WebKit based.

I'm updating the issue to reflect this.

**Comment by azuretower** [comment]
This is still an active issue

**Comment by azuretower** [comment]
This issue is unresolved

**Comment by azuretower** [comment]
Is this going to get looked at eventually?

**Comment by MahmoudAlyuDeen** [comment]
Might be time to look for alternatives 🤷‍♂️

**Comment by MahmoudAlyuDeen** [comment]
Might be time to look for alternatives 🤷‍♂️

**Comment by azuretower** [comment]
Yep. I’ve been testing out this fork. https://github.com/gtsteffaniak/filebrowser 

It seems pretty promising so far. At least someone is working on it for now.

**Comment by MahmoudAlyuDeen** [comment]
@azuretower 

> Yep. I’ve been testing out this fork.

Just deployed for testing, I think I'm staying. Folder sizes and volumes are huge wins already.

I would be hesitant to rely on such beta fork for securely and publicly access my files, but I usually disable dedicated authentication and run such services behind `NginxProxyManager` + `Authentik` for proper 2FA.

Thanks for the great reference!

**Comment by robinovitch61** [comment]
@o1egl , it seems like you are the latest regular committer/maintainer - would you be open to contributions/PRs for this issue?

**Comment by robinovitch61** [comment]
Not stale

**Comment by csuporzoltan** [comment]
Not stale, altough in recent build that button started to work for me.

**Comment by jniggemann** [comment]
Thank you for opening an issue and contributing to the filebrowser project!
I can't reproduce this on iOS/Safari 18.5 (604.1, reports as WebKit 605.1.15). Tap on the icon give me the usual green "copied to clipboard".

Can you please check if this issue persists on the latest version of Safari?

**Comment by csuporzoltan** [comment]
> Thank you for opening an issue and contributing to the filebrowser project! I can't reproduce this on iOS/Safari 18.5 (604.1, reports as WebKit 605.1.15). Tap on the icon give me the usual green "copied to clipboard".
> 
> Can you please check if this issue persists on the latest version of Safari?

I'm not the opener but I can also confirm that it does work in iOS 18.5.

**Comment by jniggemann** [comment]
> > Thank you for opening an issue and contributing to the filebrowser project! I can't reproduce this on iOS/Safari 18.5 (604.1, reports as WebKit 605.1.15). Tap on the icon give me the usual green "copied to clipboard".
> > Can you please check if this issue persists on the latest version of Safari?
> 
> I'm not the opener but I can also confirm that it does work in iOS 18.5.

Thank you very much for testing and reporting back!
I'll close this, @MahmoudAlyuDeen: Please open a new issue, if you still experience issues with WebKit based browsers.

**Comment by robinovitch61** [comment]
Confirmed fixed in 2.32.0
